src/index.html

```diff
@@ -4,9 +4,351 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#section-1" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#section-1" class="is-active nav__link">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-2" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-3" class="nav__link">SMARTPHONES</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#section-4"
+              class="nav__link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-5" class="nav__link">GADGETS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-6" class="nav__link">TABLETS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-7" class="nav__link">PHOTO</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-8" class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <article class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__photo"
+        >
+        <h1 class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</ыусешщт>
+        </h1>
```

Comment:
Try to use another semantic tag, as a rule, `h1` is used only 1 time on the page. You can use h2 or h3